### PR TITLE
show topic limits if they're there in kafka:topics

### PR DESCRIPTION
**Pull request by tcrayford**
if there are topic limits on the addon, show them in `heroku kafka:topics`

### CODE REVIEW

File: test/commands/topics_test.js [hunk 1]
```diff
@@ -43,11 +43,15 @@ describe('kafka:topics', () => {
...
+        },
         topics: []
       })
 
       return cmd.run({app: 'myapp', args: {}})
         .then(() => expect(cli.stdout).to.equal(`=== Kafka Topics on HEROKU_KAFKA_BLUE_URL
+0 / 10 topics
 
 No topics found on this Kafka cluster.
 Use heroku kafka:topics:create to create a topic.
```
Comment (uhoh-itsmaciek) [inline test/commands/topics_test.js]: This reads kinda funny with no topics: the 0/10 is mostly superfluous here--well, the zero is definitely superfluous, I suppose we may still want the ten, but I'm not sure how to integrate that. Maybe a separate formatting for this case?

```
 No topics found on this Kafka cluster.
 Use heroku kafka:topics:create to create a topic (limit 10).
```

Comment (tcrayford) [inline test/commands/topics_test.js]: fixed and squashed

Comment (tcrayford) [inline test/commands/topics_test.js]: thanks

**Comment by uhoh-itsmaciek** [comment]
Excellent, +1

**Pull request merged**